_pytest/test_server.py

```diff
@@ -54,6 +54,10 @@ def test_status(client):
         u"/parse?q=hello ńöñàśçií",
         [{u"entities": {}, u"confidence": 1.0, u"intent": u"greet", u"_text": u"hello ńöñàśçií"}]
     ),
+    ResponseTest(
+        u"/parse?q=",
+        [{u"entities": {}, u"confidence": 0.0, u"intent": None, u"_text": u""}]
```

Comment:
Just saw this, since we import `from __future__ import unicode_literals` we don't need all the `u` prefixes anymore. Different Issue?